timm/train.py

```diff
@@ -817,6 +832,8 @@ def train(config_path: str | None = None):
         )
         with open(os.path.join(output_dir, 'args.yaml'), 'w') as f:
             f.write(args_text)
+        mlflow.log_text(args_text, "config.yaml")
+        _log_params(args)
```

Comment:
I'm thinking these parameters could be something we'll handle outside of `pytorch-image-models`?

Reply:
I'd rather log them here because the config parameters and their default values are defined here